python/cudf/cudf/core/missing.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+
+
+# Pandas NAType enforces a single instance exists at a time
+# instantiating this class will yield the existing instance
+# of pandas._libs.missing.NAType, id(cudf.NA) == id(pd.NA).
+from pandas import NA  # noqa: F401
```

Comment:
Do we want to do a similar thing in https://github.com/rapidsai/cudf/blob/4913a9be73e731f0604bd9950f71d2ad1bd62399/python/cudf/cudf/core/udf/typing.py#L20 ?

Maybe we run `from pandas import NA` and then use `type(NA)` instead of importing the private `_NAType`.

Reply:
Yes! that would be much better. Added in https://github.com/rapidsai/cudf/pull/10791/commits/f63a9582e3214212da1bd87566d1effee06b9b77